### DISCUSSION TITLE
SQL command not properly ended

### DISCUSSION THREAD

**Discussion by infectormp** [dbeaver/dbeaver#246] [Q&A] [answered]
When i try execute this two string at same time i get error.

```
COMMENT ON COLUMN XXX.TABLES.SELECT_SQL IS 'ххх';
COMMENT ON COLUMN XXX.TABLES.SELECT_SQL_FORMAT IS 'ххх';
```

DBCException: SQL Error [933] [42000]: ORA-00933: SQL command not properly ended
  java.sql.SQLSyntaxErrorException: SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

**Comment by serge-rider** [comment, marked as answer]
What do you mean "at the same time"? These are two separate queries, they should be executed separately. You can execute them with al+x or ctrl+alt+x - then dbeaver will parse them and execute one by one.

**Comment by infectormp** [comment]
I select 2 lines and press ctrl-enter

**Comment by infectormp** [comment]
alt+x work for me, i thought ctrl-enter must work too because lines separate by `;`
